src/network/netdev/bridge.c

```diff
@@ -102,7 +103,7 @@ static int netdev_bridge_post_create(NetDev *netdev, Link *link, sd_netlink_mess
                         return log_netdev_error_errno(netdev, r, "Could not append IFLA_BR_PRIORITY attribute: %m");
         }
 
-        if (b->default_pvid > 0) {
+        if (b->default_pvid != VLANID_INVALID) {
```

Comment:
What I think is "none" is like disabled but here pvid/vlanid `0` is perfectly valid. Hence that makes sense here.